src/serac/numerics/equation_solver.cpp

```diff
@@ -457,9 +468,22 @@ class TrustRegion : public mfem::NewtonSolver {
   void precond(const mfem::Vector& x_, mfem::Vector& v_) const
   {
     SERAC_MARK_FUNCTION;
-    trPrecond.Mult(x_, v_);
+    tr_precond.Mult(x_, v_);
   };
 
+  /// estimate the minimum eigenvalue and its eigenvector
+  // void estimateExtremalEigenvector(const mfem::Operator& op, mfem::Vector& v, mfem::Vector& w, size_t iters, double
+  // changeTol)
+  //{
+  // CALI_CXX_MARK_FUNCTION;
+  // v /= v.Norml2();
+  // op.Mult(v, w);
+  // double lam = Dot(v, w); // first eigenvalue estimate
+  // for (size_t i=0; i < iters; ++i) {
+  //   w /= w.Norml2();
+  // }
+  //}
+
```

Comment:
```suggestion
  /// estimate the minimum eigenvalue and its eigenvector
  // void estimateExtremalEigenvector(const mfem::Operator& op, mfem::Vector& v, mfem::Vector& w, size_t iters, double
  // changeTol)
  //{
  // CALI_CXX_MARK_FUNCTION;
  // v /= v.Norml2();
  // op.Mult(v, w);
  // double lam = Dot(v, w); // first eigenvalue estimate
  // for (size_t i=0; i < iters; ++i) {
  //   w /= w.Norml2();
  // }
  //}
```

Preferred to remove dead code unless there is a reason for it

Reply:
done